### PR TITLE
AM-210 Removing sign-up errors

### PR DESCRIPTION
**Pull request by ParsaHonarmand**
Removing errors for when first and last name are more than 1 word

### CODE REVIEW

File: src/components/SignUp/index.js [hunk 1]
```diff
@@ -174,7 +172,7 @@ class SignUp extends PureComponent {
             </Button>
           </Form>
 
-          <Link to="/sign-in" bold>
+          <Link to="/sign-in" href="/sign-in" bold>
```
Comment (karimbounekhla) [inline src/components/SignUp/index.js:175]: To remove the eslint warning and avoid confusions, I would have `href="/#"`

**Review by karimbounekhla — APPROVED** [review]
Looks good to me, only one minor comment.

**Review by karimbounekhla — APPROVED** [review]

**Comment by QasimMuhmammad** [comment]
Can you close this request and open a merge into develop?

**Comment by ParsaHonarmand** [comment]
Closing to merge into develop instead

**Pull request closed without merging**